aidies-project/client/src/components/Navigation.js

```diff
@@ -1,18 +1,39 @@
 import React from 'react';
+import '../css/navigation.css';
+import logo from '../images/aidies-logo.svg';
 import { NavLink } from 'react-router-dom';
 import LogoutButton from './LogoutButton';
+import {Button} from '@material-ui/core';
+
 function Navigation(props) {
 
     return (
-        <nav>
-            <ul>
-                <li><NavLink to="/" activeClass="active">Home</NavLink></li>
-                <li><NavLink to="/users" activeClass="active">Users</NavLink></li>
-                <li>{props.needLogin 
-                 ? <NavLink to="/login" activeClass="active">Login</NavLink> 
-                 : <LogoutButton/>} </li>
-                
-            </ul>
+        <nav className="nav-main">
+            <div className="navigation-container">
+                <div className="nav-button">
+                    <div className="nav-icon"><i className="fas fa-plus"></i></div>
+                    <NavLink to="/campaign/create" activeClassName="active"><Button>START A CAMPAIGN</Button></NavLink>
+                </div>
+                <div className="logo"><NavLink to={`/`}>
+                    <img
+                        className="nav-image"
+                        src={logo}
+                        alt='website logo'
+                        width="125"
+                    />
+                </NavLink></div>
+                <div className="nav-right-buttons">
+                    <div className="nav-button">
+                        <div className="nav-icon"><i className="fas fa-search"></i></div>
+                        <NavLink to="/search" activeClassName="active"><Button>SEARCH</Button></NavLink>
+                    </div>
+                    <div className="nav-button right" >
+                        <div className="nav-icon"><i className="fas fa-sign-in-alt"></i></div>
+                    {props.needLogin 
+                            ? <NavLink to="/login" activeClassName="active"><Button>LOGIN</Button></NavLink> 
+                    : <LogoutButton/>} </div>
+                </div>
```

Comment:
You should check out setting up ESLint for your project, so that you get style-guide warnings. You can set it up with the AirBnB JavaScript style guide!

https://medium.com/@Tunmise/set-up-eslint-with-airbnb-style-guide-in-5-minutes-d7b4cc5707f8
https://github.com/airbnb/javascript

You can add `"linter": "eslint './**/*.js' --fix"` to your `client` folder's `package.json` and then run `npm run linter` to get the ESLint warnings! Feel free to reference this example:

**.eslintrc.json**
```json
{
    "env": {
        "browser": true,
        "es6": true
    },
    "extends": [
        "plugin:react/recommended",
        "airbnb"
    ],
    "globals": {
        "Atomics": "readonly",
        "SharedArrayBuffer": "readonly",
        "mapboxgl": "readonly",
        "$": "readonly"
    },
    "parserOptions": {
        "ecmaFeatures": {
            "jsx": true
        },
        "ecmaVersion": 2018,
        "sourceType": "module"
    },
    "plugins": [
        "react"
    ],
    "rules": {
        "arrow-parens": 0,
        "react/forbid-prop-types": 0
    }
}
```